index.html

```diff
@@ -268,10 +268,15 @@ <h1 id="skill-text">Skills</h1>
             </p>
             <form class="form" action="https://formspree.io/f/xrgjdpow" method="POST">
                 <input type="text" name="name" class="text" placeholder="Full name" maxlength="30" required />
+
+               <div class="email">
+                <input id="mail" type="email" class="text" placeholder="Email address" required />
+                </div>
                 <input type="email" class="text" placeholder="Email address" pattern=" (?=.*[a-z])(?=.*[A-Z])"  required />
```

Comment:
- [ ] Kindly remove this line because it is make unexpected behavior and cause validation not working